cp-remote

```diff
@@ -335,12 +336,34 @@ function port_forward {
     kubectl --context="$(context)" --namespace="$(namespace)" port-forward "$POD" "$LOCAL_PORT":"$REMOTE_PORT"
 }
 
+function check_version {
+
+    CURRENT_VERSION=`cat ${0} | grep "^# Version" | head -1 | awk '{print $3}'`
+    LATEST_VERSION=`curl -s https://raw.githubusercontent.com/continuouspipe/remote-environment-client/gh-pages/downloads/cp-remote-latest | grep "# Version" | awk '{print $3}'`
```

Comment:
It does need that, I've added it now. 

@sroze not especially as that does not say what the latest versions is, I would need to extract all the version numbers and them sort them correctly. The sort utility of osx does not have the -V option that sorts version numbers so this would be a difficult.